themes/ddbasic/template.php

```diff
@@ -397,6 +397,25 @@ function ddbasic_preprocess_node(&$variables, $hook) {
 
         $variables['content']['group_right_col_search']['more_link'] = $more_link;
         break;
+
+      case 'ding_page':
+        $more_link = array(
+          '#theme' => 'link',
+          '#text' => t('Read more'),
+          '#path' => 'node/' . $variables['nid'],
+          '#options' => array(
+            'attributes' => array(
+              'title' => $variables['title'],
+            ),
+            'html' => FALSE,
+          ),
+          '#prefix' => '<span class="page-link">',
+          '#surfix' => '</span>',
+          '#weight' => 6,
+        );
+
```

Comment:
[Apparently](https://www.drupal.org/node/1187032#comment-4711898) it's better to use '#type' => 'link' as otherwise some l() preprocessing gets missed. The syntax is valid, though, and used elsewhere in the code.